documentation/src/main/asciidoc/topics/proc_configuring_custom_merge_policies.adoc

```diff
@@ -0,0 +1,68 @@
+[id="configuring-custom-merge-policies_{context}"]
+= Configuring custom merge policies
+
+If you implement the `EntryMergePolicy` API to create a custom merge policy to handle network partitions.
+
+.Prerequisites
+
+* Implement the `EntryMergePolicy` API.
++
+[source,java,options="nowrap",subs=attributes+]
+----
+include::code_examples/CustomMergePolicyEntryMerge.java[]
+----
+
+.Procedure
+
+. Deploy your merge policy implementation to {brandname} Server if you use remote caches.
+.. Package your classes as a JAR file that includes a `META-INF/services/org.infinispan.conflict.EntryMergePolicy` file that contains the fully qualified class name of your merge policy.
++
+----
+# List implementations of EntryMergePolicy with the full qualified class name
+org.example.CustomMergePolicy
+----
+.. Add the JAR file to the `server/lib` directory.
++
+[TIP]
+====
+Use the `install` command with the {brandname} Command Line Interface (CLI) to download the JAR to the `server/lib` directory.
+====
+. Open your {brandname} configuration for editing.
+. Configure cache encoding with the `encoding` element or `encoding()` method as appropriate.
++
+For remote caches, if you use only object metadata for comparison when merging entries then you can use `application/x-protostream` as the media type. In this case {brandname} returns entries to the `EntryMergePolicy` as `byte[]`.
++
+If you require the object itself when merging conflicts then you should configure caches with the `application/x-java-object` media type. In this case you must deploy the relevant ProtoStream marshallers to {brandname} Server so it can perform `byte[]` to object transformations if clients use Protobuf encoding.
```

Comment:
@ryanemerson Hey, I moved that detail about encoding with remote caches and custom merge policies as part of the procedure. This is pretty important info imo and should be highlighted as a distinct step. Hopefully I didn't garble any technical details though. Could you give it a look for accuracy? Cheers.

Reply:
That looks really good :+1: